vllm/model_executor/layers/quantization/utils/w8a8_utils.py

```diff
@@ -207,3 +223,26 @@ def apply_int8_linear(
                                  scale_b=weight_scale,
                                  out_dtype=input.dtype,
                                  bias=bias)
+
+
+def convert_to_e4m3fnuz(
```

Comment:
Can we rename `convert_to_e4m3fnuz` to `normalize_ocp_to_e4m3fnuz`?

Reply:
> @charlifu Can we rename `convert_to_e4m3fnuz` to `normalize_ocp_to_e4m3fnuz`?

I am concerned that mentioning ocp will cause confusion here, since "ocp" is never in the context.

Reply:
OCP is Open Compute Project, its FP8 can be found here [OCP 8-bit Floating Point Specification](https://www.opencompute.org/documents/ocp-8-bit-floating-point-specification-ofp8-revision-1-0-2023-12-01-pdf-1)
You can either add a comment on OCP if use `normalize_ocp_to_e4m3fnuz`, or simply use `normalize_e4m3fn_to_e4m3fnuz` (intention is to differentiate with direct data conversion between them).

Reply:
I like `normalize_e4m3fn_to_e4m3fnuz`, although feel free to leave the link to the specification in a comment